drivers/staging/vc04_services/vchiq-mmal/mmal-parameters.h

```diff
@@ -780,7 +836,101 @@ struct mmal_parameter_camera_info_t {
 	struct mmal_parameter_camera_info_camera_t
 		cameras[MMAL_PARAMETER_CAMERA_INFO_MAX_CAMERAS];
 	struct mmal_parameter_camera_info_flash_t
-				flashes[MMAL_PARAMETER_CAMERA_INFO_MAX_FLASHES];
+		flashes[MMAL_PARAMETER_CAMERA_INFO_MAX_FLASHES];
+};
+
+struct mmal_parameter_ccm_t {
```

Comment:
I know I'm guilty above, but kernel style normally doesn't include ```_t``` suffixes. They're not typdef'ed, therefore you have the ```struct``` telling you the type anyway.

Reply:
Yeah, I wanted to keep the format consistent with the rest of the file.  Should I take the hit and just fixup all the _t usage, or just fixup my additions?

Reply:
Most of the file is without the _t. It's only the last couple that have it.
Fix them up as a separate patch, then add your new controls.

Reply:
I'll add the_t fixes to the same patch that fixes whitespace in the file before adding the new controls then.